builds/misc/images.yaml

```diff
@@ -184,26 +184,26 @@ jobs:
 ################################################################################
 # linux - Rust amd64
 ################################################################################
-  - template: templates/image-linux-rust-jobs.yaml
-    parameters:
-      archName: amd64
-      targetArch: x86_64
+  # - template: templates/image-linux-rust-jobs.yaml
```

Comment:
Ignore all changes in this file.  This is just to unblock image build for testing.